src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,147 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_LENGTH = 1 << 4;
+    static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int actualLoadFactorNumber;
+    private int size;
+    private int newLength;
+    private Node<K, V>[] table;
 
     @Override
     public void put(K key, V value) {
-
+        Node<K, V>[] oldTab = table;
+        if (table == null || size >= actualLoadFactorNumber) {
```

Comment:
initialize table in constructor, you should not have situation when it's null

Reply:
Not fixed

Reply:
?

Reply:
i did not send any requests one more time